src/PrestaShopBundle/Entity/Shipment.php

```diff
@@ -0,0 +1,260 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShopBundle\Entity;
+
+use DateTime;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table()
+ *
+ * @ORM\Entity()
+ */
+class Shipment
+{
+    /**
+     * @ORM\Id
+     *
+     * @ORM\Column(name="id_shipment", type="integer")
+     *
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private int $id;
+
+    /**
+     * @ORM\Column(name="id_order", type="integer")
+     */
+    private int $orderId;
+
+    /**
+     * @ORM\Column(name="id_carrier", type="integer")
+     */
+    private int $carrierId;
+
+    /**
+     * @ORM\Column(name="id_delivery_address", type="integer")
+     */
+    private int $deliveryAddressId;
+
+    /**
+     * @ORM\Column(name="shipping_cost_tax_excl", type="float")
+     */
+    private float $shippingCostTaxExcluded;
+
+    /**
+     * @ORM\Column(name="shipping_cost_tax_incl", type="float")
+     */
+    private float $shippingCostTaxIncluded;
+
+    /**
+     * @ORM\Column(name="packed_at", type="datetime", nullable=true)
+     */
+    private ?DateTime $packedAt;
+
+    /**
+     * @ORM\Column(name="shipped_at", type="datetime", nullable=true)
+     */
+    private ?DateTime $shippedAt;
+
+    /**
+     * @ORM\Column(name="delivered_at", type="datetime", nullable=true)
+     */
+    private ?DateTime $deliveredAt;
+
+    /**
+     * @ORM\Column(name="tracking_number", type="string", nullable=true)
+     */
+    private ?string $trakingNumber;
+
+    /**
+     * @ORM\Column(name="created_at", type="datetime", nullable=false)
```

Comment:
As discussed in previous comment, we should follow the legacy convention for this column (only on the DB level)
```suggestion
     * @ORM\Column(name="date_add", type="datetime", nullable=false)
```